docs/docs.css

```diff
@@ -154,3 +152,101 @@ button.active {
     overflow-x: scroll;
   }
 }
+
+footer {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px;
+  border-top: 2px solid white;
+  background-color: #dfdfdf;
+}
+
+footer button {
+  background-color: #dfdfdf;
+}
+
+footer button img {
+  width: 71px;
+  height: 18px;
+  padding-top: 2px;
+}
+
+.menu-bar {
+  padding: 0;
+  position: relative;
+  display: inline-block;
+}
+
+.menu-bar .menu-bar-content {
+  display: none;
+  position: absolute;
+  min-width: 160px;
+  box-shadow: 2px 2px 2px 0px rgb(0, 0, 0);
+  padding: 0;
+  z-index: 1;
+  bottom: 21px;
+  left: 3px;
+  border-left: 3px solid white;
+  border-top: 3px solid white;
+  background-color: #dfdfdf;
+}
+
+.menu-bar .menu-bar-content .side {
+  writing-mode: tb-rl;
+  transform: rotate(180deg);
+  align-items: center;
+  display: flex;
+  padding: 5px 5px 0px 5px;
+  font-size: 20px;
+  min-height: 260px;
+  flex-basis: 20%;
+  color: white;
+  background: linear-gradient(45deg, #000014, navy, #1084d0);
+  margin: 0px 0px 3px 0px;
+}
+
+.font-weight-bold {
+  font-weight: bold;
+}
+
+.menu-bar .menu-bar-content .main {
+  flex-grow: 1;
+  display: flex;
+}
+
+.menu-bar .menu-bar-content .main .items {
+  list-style-type: none;
+  padding: 0;
+  width: 100%;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.menu-bar .menu-bar-content .main .items .item {
+  padding: 10px 10px 10px 18px;
+  min-width: 200px;
+  ;
+}
+
+.menu-bar .menu-bar-content .main .items .item:hover {
```

Comment:
Hi! I am unlikely to merge this for accessibility reasons. 

Primarily, `:hover` means I won't be able to use this menu with a keyboard. Additionally, we'll need to let screen readers know when the menu is open via `aria-expanded`.

https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton

Please continue exploring this idea outside of this repo though, I'd love to see what you build. We'll continue exploring this in #80.

Reply:
thanks bro for helping me, i have question:
can i use javascript in this project?
if i can use, i want write like this code for `aria-expanded`.
`  function toggle(id, id2) {
    var n = document.getElementById(id);
	if (n.style.display != 'none') 
	  {
	  n.style.display = 'none';
      document.getElementById(id2).setAttribute('aria-expanded', 'false');
  }
  else
  {
  n.style.display = '';
  document.getElementById(id2).setAttribute('aria-expanded', 'true');
	  }
  }`